CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * [ENHANCEMENT] Dashboards: adjust layout of "rollout progress" dashboard panels so that the "rollout progress" panel doesn't require scrolling. #5113
 * [ENHANCEMENT] Dashboards: show container name first in "pods count per version" panel on "rollout progress" dashboard. #5113
 * [BUGFIX] Dashboards: fix "unhealthy pods" panel on "rollout progress" dashboard showing only a number rather than the name of the workload and the number of unhealthy pods if only one workload has unhealthy pods. #5113
+* [BUGFIX] Dashboards: fix "unhealthy pods" panel on "rollout progress" dashboard extending below the bottom of other dashboard panels. #5200
```

Comment:
[nit] Given this fixes an issue introduced in #5113 you can just add the PR number to that CHANGELOG entry.

```suggestion
* [BUGFIX] Dashboards: fix "unhealthy pods" panel on "rollout progress" dashboard showing only a number rather than the name of the workload and the number of unhealthy pods if only one workload has unhealthy pods. #5113 #5200
```